.gitignore

```diff
@@ -1,3 +1,4 @@
 /node_modules/
 /components/
-.bowerrc
\ No newline at end of file
+.bowerrc

```

Comment:
Put your bowerrc inside the commit
remove components in gitignore
I should have committed the .bowerrc file before